script.module.kodi-six/libs/kodi_six/utils.py

```diff
@@ -86,13 +90,12 @@ def patch_object(obj):
         obj = _wrap_class(obj)
         for memb_name, member in inspect.getmembers(obj):
             # Do not patch special methods!
-            if (inspect.ismethoddescriptor(member) and
-                    not memb_name.endswith('__')):
+            if (inspect.ismethoddescriptor(member) and not memb_name.endswith('__')):
```

Comment:
Please respect PEP8 line lengths limitation.

Reply:
Well, the PEP8 line-length is contested these days, no sane projects still sticks to 80 columns. People have wide monitors these days. In any case, how the line was split caused sanity issues.

Reply:
This project does follow 80 columns limitation. Please restore those strings as they were since this change has nothing to do with the goal of this PR.